openhands/core/config/llm_config.py

```diff
@@ -80,6 +80,7 @@ class LLMConfig(BaseModel):
     # Note: this setting is actually global, unlike drop_params
     modify_params: bool = Field(default=True)
     disable_vision: bool | None = Field(default=None)
+    disable_stop_word: bool | None = Field(default=None)
```

Comment:
Let's just default it to false?

Reply:
Changed in [005674c](https://github.com/All-Hands-AI/OpenHands/pull/10274/commits/005674c18539308ccce22a7994f58e497696379d)